onadata/libs/serializers/data_serializer.py

```diff
@@ -110,6 +111,10 @@ def to_representation(self, instance):
         ret = super(DataInstanceSerializer, self).to_representation(instance)
         if 'json' in ret:
             ret = ret['json']
+            # Remove metadata fields from the instance
+            for field in METADATA_FIELDS:
+                if field in ret:
+                    del ret[field]
```

Comment:
Could we `exclude` these fields ?

Reply:
Might need to move this to an entirely new serializer as this serializer is also used for the data endpoint.
I'll change this

Reply:
Actually, I dont believe we can use `excclude` here
As we need to remove some of the fields we get from the json we receive for a specific submission
Exclude would have worked if this was removing some of the fields from the model.
Correct me if am wrong?

Reply:
Mhh yea, thats true